pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -390,6 +395,17 @@ var supportedReinvocationPolicies = sets.NewString(
 	string(admissionregistration.IfNeededReinvocationPolicy),
 )
 
+var supportedValidationPolicyReason = sets.NewString(
+	string(metav1.StatusReasonUnauthorized),
+	string(metav1.StatusReasonForbidden),
+	string(metav1.StatusReasonNotFound),
+	string(metav1.StatusReasonInvalid),
+	string(metav1.StatusReasonTooManyRequests),
+	string(metav1.StatusReasonRequestEntityTooLarge),
+	string(metav1.StatusReasonConflict),
+	string(metav1.StatusReasonInternalError),
```

Comment:
I think we should drop StatusReasonInternalError... if we successfully evaluate their expression and it rejects the request, they need to give us a non 5xx error to return to the user

we will still return 5xx errors if the CEL evaluation hits an actual error, but that is independent of the configured reason they get to specify in their policy for normal rejections based on successful evaluations